src/io/inputs/model-converter/include/antares/io/inputs/model-converter/modelConverter.h

```diff
@@ -24,6 +24,8 @@
 
 #include "antares/io/inputs/yml-model/Library.h"
 
+#include "ForbiddenNodes.h"
+
```

Comment:
It seems that header modelConverter.h does not make use of ForbiddenNodes.h (implementation detail).

So I suggest including it only in **src/io/inputs/model-converter/modelConverter.cpp**